README.md

```diff
@@ -40,7 +40,7 @@ The production build is minified and does not generate console warnings. It will
 | Debugging enabled | <p align="center">✓</p> |  |
 | Minified |  | <p align="center">✓</p> |
 | Latest release (*may be hotlinked if desired*) | [jquery-migrate-3.5.2.js](https://code.jquery.com/jquery-migrate-3.5.2.js) | [jquery-migrate-3.5.2.min.js](https://code.jquery.com/jquery-migrate-3.5.2.min.js) |
-| \* Latest work-in-progress build | [jquery-migrate-git.js](https://releases.jquery.com/git/jquery-migrate-git.js) | [jquery-migrate-git.min.js](https://releases.jquery.com/git/jquery-migrate-git.min.js) |
+| \* Latest work-in-progress build | [jquery-migrate-3.x-git.js](https://releases.jquery.com/git/jquery-migrate-3.x-git.js) | [jquery-migrate-3.x-git.min.js](https://releases.jquery.com/git/jquery-migrate-3.x-git.min.js) |
```

Comment:
This is a little confusing. The latest work in progress build is 4.x, but I understand this the latest work in progress build on the 3.x branch. I'm not sure a change is needed, but I'll mention it anyway.

Reply:
Makes sense. I wanted to give people a way to experiment with 3.x prereleases since we'll have to maintain the 3.x branch as long as we support jQuery 3.x at least.

Reply:
I like `Latest work-in-progress 3.x build`

Reply:
Done.